src/main/index.ts

```diff
@@ -953,7 +957,7 @@ const router = t.router({
     if (!PRIVILEDGED_LAUNCHER_WINDOWS)
```

Comment:
Minor, but this is actually misspelt

```suggestion
    if (!PRIVILEGED_LAUNCHER_WINDOWS)
```

Reply:
thanks :+1: